### ISSUE TITLE
The repo is almost 1GB

### ISSUE THREAD

**Issue by rogerdahl** [Moonbase59/loudgain#47]
I see that it's because of a bunch of commits of the static binary, each at 26-27MB. Might be worth fixing even though it will require a force push, e.g., with BFG Repo Cleaner. You can at least prevent the repo from getting even larger by moving the binary out and attaching it to a GitHub tag (which is the normal way to deal with frequently changing binaries in GitHub).

**Comment by JuniorJPDJ** [comment]
Yup. Binaries should be moved away from repo and added to github releases. It will also be clear what version the binary is representing.

**Comment by mavit** [comment]
Duplicate of #26.
